test_eradicate.py

```diff
@@ -137,11 +137,14 @@ def test_comment_contains_code_with_encoding(self):
             '# coding=utf-8'))
 
         self.assertFalse(eradicate.comment_contains_code(
-            '#coding = utf-8'))
+            '#coding= utf-8'))
```

Comment:
I'm not sure what this change is for.

Reply:
Thank you for reviewing.

The change is to a valid PEP 263 encoding style.
`#coding = utf-8` does not considered as an encoding definition according to PEP 263 (`^[ \t\f]*#.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)`).